### PR TITLE
Compact shotgun re-added

### PR DESCRIPTION
**Pull request by LukasBeedellCodestuff**
## About The Pull Request

This pr seeks to re-add the compact shotgun (slightly buffed with 1 more ammo) and buff up its larger brother the combat shotgun (with 2 more ammo.) 

## Why It's Good For The Game
With the recent laser buffs, there is a real possibility for the compact shotgun to return as a unique weapon to make the HOS slightly more powerful. I am aware that it was a warden's weapon previously but the HoS doesn't really have many fun toys to play with. The warden already has crav maga (100x cooler than the laser) so giving this beast to the HOS could help make it a more attractive and powerful head to play. (Given 1 extra shot to keep up with the crazy lasers nowadays.) 

In regards to the slight combat shotgun buff. The gun itself is ass, it's barely ever used and the riot shotgun is superior because you can actually put it in your armour slot. The hope is that this buff will make people actually use it because it carries a lot of shots now so the viability may increase. 

## Changelog
:cl:
add: Added compact shotgun to the hos locker
add: Added compact shotgun as a traitor objective 
balance: gives the compact shotgun 1 extra shot
/:cl:

### CODE REVIEW

**Comment by Jackal-boop** [comment]
wasnt the compact shotgun a warden thing?

**Comment by LukasBeedellCodestuff** [comment]
> wasnt the compact shotgun a warden thing?

It was previously, yes.

**Comment by Jackal-boop** [comment]
the HOS already has the multiphase, why are you giving it to them insntead of the warden? or putting it in the armoury

**Comment by carlarctg** [comment]
power crep

**Comment by LukasBeedellCodestuff** [comment]
> power crep

Lasers are king ATM.....

**Comment by LukasBeedellCodestuff** [comment]
> the HOS already has the multiphase, why are you giving it to them insntead of the warden? or putting it in the armoury

the warden sucks. The HoS having 2 guns is cool because he can have on in his armour and one in his bag.

**Comment by san7890** [comment]
> You currently have a negative Fix/Feature pull request delta of -3. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

sorry gamer get your GBP up before you start doing feature/balance PRs. there's probably a few easy bugs here and there that you can fix, i'll remove the DNM as soon as you open at least one fix PR to offset this delta

**Comment by Jackal-boop** [comment]
> > the HOS already has the multiphase, why are you giving it to them insntead of the warden? or putting it in the armoury
> 
> the warden sucks. The HoS having 2 guns is cool because he can have on in his armour and one in his bag. Warden already has the fantastic CQC gloves, I think a shotgun might be a bit too much for it as it's meant to be a support-y kinda role.

the hos already has 2 start guns, the egun and multiphase

**Comment by LukasBeedellCodestuff** [comment]
> > > the HOS already has the multiphase, why are you giving it to them insntead of the warden? or putting it in the armoury
> > 
> > 
> > the warden sucks. The HoS having 2 guns is cool because he can have on in his armour and one in his bag. Warden already has the fantastic CQC gloves, I think a shotgun might be a bit too much for it as it's meant to be a support-y kinda role.
> 
> the hos already has 2 start guns, the egun and multiphase

Well the EGUN I don't really count as one tbqh, it's not special or unique.

**Comment by Pickle-Coding** [comment]
I don't think security should be given ballistics. It's really unhealthy for station maintenance.

File: code/__DEFINES/inventory.dm [hunk 1]
```diff
@@ -181,6 +181,7 @@ GLOBAL_LIST_INIT(detective_vest_allowed, list(
 	/obj/item/storage/belt/holster/detective,
 	/obj/item/storage/belt/holster/nukie,
 	/obj/item/storage/belt/holster/energy,
+	/obj/item/gun/ballistic/shotgun/automatic/combat/compact
```
Comment (Fikou) [inline code/__DEFINES/inventory.dm:184]: leave trailing commas at the end of lists

File: code/game/gamemodes/objective_items.dm [hunk 2]
```diff
@@ -216,6 +216,16 @@
 /obj/item/gun/energy/e_gun/hos/add_stealing_item_objective()
 	return add_item_to_steal(src, /obj/item/gun/energy/e_gun/hos)
 
+/datum/objective_item/steal/compactshotty
```
Comment (Fikou) [inline code/game/gamemodes/objective_items.dm:219]: are you sure you wanna add this? this is just for antags like wizard or bb and its bulky, which makes it hard to store (this is we we buffed antique laser and hos gun to be normal sized)

**Review by Fikou — CHANGES_REQUESTED** [review]

File: code/game/gamemodes/objective_items.dm (hunk 2 above)
Comment (LukasBeedellCodestuff) [reply code/game/gamemodes/objective_items.dm:219]: > are you sure you wanna add this? this is just for antags like wizard or bb and its bulky, which makes it hard to store (this is we we buffed antique laser and hos gun to be normal sized)

Maybe I'll change it to destory.

**Comment by LemonInTheDark** [comment]
> sorry gamer get your GBP up before you start doing feature/balance PRs. there's probably a few easy bugs here and there that you can fix, i'll remove the DNM as soon as you open at least one fix PR to offset this delta

He's a headmin he's burned out already

**Comment by SomeoneYouProbablyKnow** [comment]
"to the hos locker" unbiased PR moment

**Comment by Mothblocks** [comment]
> With the recent laser buffs, there is a real possibility for the compact shotgun to return as a unique weapon to make the HOS slightly more powerful

power creep

**Comment by Vect0r2** [comment]
> > With the recent laser buffs, there is a real possibility for the compact shotgun to return as a unique weapon to make the HOS slightly more powerful
> 
> 
> 
> power creep

Wasn't a part of the laser buff to give room to buff other options? I might be misinformed about it, but that is what me a other players assumed.
Also, give it to the warden man.

**Comment by san7890** [comment]
> He's a headmin he's burned out already

a majority of recent player headmins have an abundance of gbp i have a keen interest in ensuring the trend continues into the far future

**Comment by wesoda25** [comment]
iirc the mime used to spawn with these, so you should probably put it in the toy chest

**Comment by atlasle** [comment]
> sorry gamer get your GBP up before you start doing feature/balance PRs. there's probably a few easy bugs here and there that you can fix, i'll remove the DNM as soon as you open at least one fix PR to offset this delta

Funny that you assume that Lukas knows how to code

**Comment by SimplySmiley123** [comment]
> I don't think security should be given ballistics. It's really unhealthy for station maintenance.

Ah yes and everyone should have only lasers to make it extremely easy for an antagonist to counter them without much effort whatsoever.
De-sword became the meta considering shotguns are useless nowadays and WTs are locked behind a traitor emagging the cargo console. Especially with nukies it's cancerous as they justn eed a shield module and a de-sword to become nigh unkillable unless someone throws a body or gambles with their soon to be decapitated head to try and tackle them.

**Comment by optimumtact** [comment]
remove the combat shotgun buff to another PR please

**Comment by optimumtact** [comment]
The rest of the PR is approved as a reward for keith4's headmin term.

**Comment by dieamond13** [comment]
give it to the warden

**Comment by nikothedude** [comment]
god is dead

**Comment by ju45he** [comment]
Krav maga has only 2 useful abilities, the disarming mechanic is never used since you could more easily just knock them down to steal their item and shoe them, the breathless punch is extra shitty.
The mute punch and knockdown are also pretty shitty since the mute punch requires you to melee a magical person or someone with a gimmicky voice sensor. 
The knockdown is worse than the tele-batong in every way except the tele-batong can be disarmed if they let them selves get knocked into a wall.
Shotguns are also shit, and never used except by tiders and as a last option.
Lasers are cooler and you can find ammo for them almost anywhere.
Shotguns are going to continue to be shit until tech shells can be printed without the need to physically craft them, and when they are they would get nerfed instantly.

**Comment by SomeoneYouProbablyKnow** [comment]
Sir this is a Wendy's

**Pull request merged**